webpack.dev.js

```diff
@@ -7,6 +7,10 @@ var CopyWebpackPlugin = require('copy-webpack-plugin');
 var DashboardPlugin = require('webpack-dashboard/plugin');
 
 var config = {
+  mode: 'development',
+  devServer: {
+    port: 3735
```

Comment:
Looking at the old dev server config, should this also set `host: process.env.HOST || "localhost"`?

Reply:
Yeah I can add that. I'm not sure if anyone was using it before, but let's keep parity anyway.

Reply:
I'm not sure if anyone was using it either. At least, I don't see it in the start scripts anywhere.